railroad/cargo_wagon.rb

```diff
@@ -3,7 +3,23 @@
 class CargoWagon < Wagon
   attr_reader :type
   
-  def initialize
+  VOLUME = 150
+
+  def initialize(volume = VOLUME)
     @type = 'cargo'
+    @all_volume = volume
+    @volume = volume
```

Comment:
```suggestion
    @free_volume = volume
```
чем более явное именование, тем меньше шансов наделать ошибок и тем проще читать и понимать код